core-test/src/test/scala/org/bitcoins/core/currency/CurrencyUnitSpec.scala

```diff
@@ -67,6 +72,21 @@ class CurrencyUnitSpec extends Properties("CurrencyUnitSpec") {
     else Try(num1 * num2).isFailure
   }
 
+  property("Multiply a satoshi value with an int") = Prop.forAll(
+    satoshiWithInt
+  ) {
+    case (sat, int) =>
+      val safeProduct = sat.multiplySafe(int)
+      val underlyingProduct = sat.toBigInt * int
+      if (underlyingProduct < Satoshis.max.toBigInt && underlyingProduct > Satoshis.min.toBigInt) {
+        assert(safeProduct.isSuccess)
```

Comment:
I don't think this line is necessary, and isn't the `assert` you are familiar with in scalatest. 

Just a note, these tests are run using scalacheck not scalatest. Scalatest has a dependency on scalacheck, which means you can do this stuff in scalatest too -- which is what I am trying to move towards in #335